src/llmq/quorums_signing_shares.cpp

```diff
@@ -613,24 +613,20 @@ void CSigSharesManager::CollectPendingSigSharesToVerify(
         }
     }
 
-    {
-        LOCK(cs_main);
```

Comment:
This triggered a cs_main -> quorumsCacheCs lock sequence, while another thread did the reverse.

Confirmed that this lock was removed from Dash as well.

Reply:
great fine.